### ISSUE TITLE
At least two variables have the same name: FullyConnected/W

### ISSUE THREAD

**Issue by Kishwar** [tflearn/tflearn#1020]
I am trying to run tflearn.DNN(net) on net below. I am getting error "At least two variables have the same name: FullyConnected/W".

# Network building
net = tflearn.input_data([None, width, height])
net = tflearn.lstm(net, 128, dropout=0.8)
net = tflearn.fully_connected(net, classes, activation='softmax')
net = tflearn.regression(net, optimizer='adam', learning_rate=learning_rate, loss='categorical_crossentropy')

**Comment by Kishwar** [comment]
Yes. Change tf version to 1.2 and python 2.7.

**Comment by DhanushDK17** [comment]
Thanks.

**Comment by vsauvage-ciis** [comment]
> Yes. Change tf version to 1.2 and python 2.7.
Aren't these old version of python and TensorFlow?
I have the same issue running tf 1.5.0 and python 3.6

**Comment by zhvankh** [comment]
You can try to delete this part:

### add this "fix" for tensorflow version errors
col = tf.get_collection(tf.GraphKeys.TRAINABLE_VARIABLES)
for x in col:
    tf.add_to_collection(tf.GraphKeys.VARIABLES, x )

**Comment by ZahinAbrar** [comment]
I faced a similar problem, one solution which seemed really cool and easy is to restarting the Kernel.
